packages/components/src/DraftConfigurator/OptionGroup/index.js

```diff
@@ -66,13 +66,19 @@ const OptionGroup = (props) => {
 
   return (
     <>
-      {props.options.map((name) => {
+      {props.options.map((nameOrGroup) => {
         let output = []
-        if (typeof name === 'object') {
+        if (typeof nameOrGroup === 'object') {
+          let group = nameOrGroup
           // Subgroup
-          for (let subGroup of Object.keys(name)) {
+          for (let subGroup of Object.keys(group)) {
             let children = []
-            for (let option of name[subGroup]) children.push(renderOption(option, true))
+            for (let name of group[subGroup]) {
+              let option = props.config.options[name]
+              if (!(option.hide instanceof Function) || !option.hide(props.pattern.settings.options)) {
```

Comment:
While the goal is to hide options when they're not needed, is it just me or is having an (optional) `show()` method on the option more intuitive than a `hide()` method?

It would be the same thing, just returning false if it should be hidden. But just `return false from show() to not show` instead of `return true from hide() to hide`.

There's obviously no right or wrong. Just thinking out loud.

Reply:
I don't have a strong preference and think show looks great. Changed.

My original reason for using hide is that the default is to show. So any show function that is defined is only there to return false. So the explanation of when to implement show is when you want to hide something. Somehow that felt backward to me.